views/room/roomTemplate.pug

```diff
@@ -23,12 +23,23 @@ mixin roomTemplate(room, group)
             span &nbsp;#{format(group.startDate, 'HH:mm')}-#{format(group.endDate, 'HH:mm')}
 
-    div(class='flex flex-row mt-3 space-x-4 sm:mt-0')
-      a(href=`/groups/new/?roomId=${room}` class='flex flex-row px-3 space-x-2 bg-orange-300 hover:bg-orange-400 dark:bg-purple-400 dark:hover:bg-purple-500 animate-hover btn')
-        svg(xmlns='http://www.w3.org/2000/svg' fill='none' viewbox='0 0 24 24' stroke='black' class='w-6 h-6 dark:text-gray-50')
-          path(stroke-linecap='round' stroke-linejoin='round' stroke-width='2' d='M9 5H7a2 2 0 00-2 2v12a2 2 0 002 2h10a2 2 0 002-2V7a2 2 0 00-2-2h-2M9 5a2 2 0 002 2h2a2 2 0 002-2M9 5a2 2 0 012-2h2a2 2 0 012 2m-6 9l2 2 4-4')
-        span(class='text-black') Foglalás
-      a(href=`/rooms/${room}` class='flex flex-row px-3 space-x-2 bg-yellow-300 hover:bg-yellow-400 dark:bg-blue-400 dark:hover:bg-blue-500 animate-hover btn')
-        svg(xmlns='http://www.w3.org/2000/svg' fill='none' viewbox='0 0 24 24' stroke='black' class='w-6 h-6')
-          path(stroke-linecap='round' stroke-linejoin='round' stroke-width='2' d='M8 7V3m8 4V3m-9 8h10M5 21h14a2 2 0 002-2V7a2 2 0 00-2-2H5a2 2 0 00-2 2v12a2 2 0 002 2z')
-        span(class='text-black') Naptár
+    div(class='flex flex-col')
+      div(class='mt-3 flex space-x-4 sm:mt-0 sm:justify-end')
+        a(href=`/groups/new/?roomId=${room}` class='flex flex-row px-3 space-x-2 bg-orange-300 hover:bg-orange-400 dark:bg-purple-400 dark:hover:bg-purple-500 animate-hover btn')
+          svg(xmlns='http://www.w3.org/2000/svg' fill='none' viewbox='0 0 24 24' stroke='black' class='w-6 h-6 dark:text-gray-50')
+            path(stroke-linecap='round' stroke-linejoin='round' stroke-width='2' d='M9 5H7a2 2 0 00-2 2v12a2 2 0 002 2h10a2 2 0 002-2V7a2 2 0 00-2-2h-2M9 5a2 2 0 002 2h2a2 2 0 002-2M9 5a2 2 0 012-2h2a2 2 0 012 2m-6 9l2 2 4-4')
+          span(class='text-black') Foglalás
+        a(href=`/rooms/${room}` class='flex flex-row px-3 space-x-2 bg-yellow-300 hover:bg-yellow-400 dark:bg-blue-400 dark:hover:bg-blue-500 animate-hover btn')
+          svg(xmlns='http://www.w3.org/2000/svg' fill='none' viewbox='0 0 24 24' stroke='black' class='w-6 h-6')
+            path(stroke-linecap='round' stroke-linejoin='round' stroke-width='2' d='M8 7V3m8 4V3m-9 8h10M5 21h14a2 2 0 002-2V7a2 2 0 00-2-2H5a2 2 0 00-2 2v12a2 2 0 002 2z')
+          span(class='text-black') Naptár
+
+      div(class='my-2 ml-2 flex flex-wrap')
+        div(class='flex items-center justify-between')
+          span(class='my-2 mr-2') foglaltság:
+        div(class='flex items-center justify-between')
+          each dayData in weekData
+            div(class='flex flex-col mx-1 text-center text-sm font-light')
+              - GREENS = ["#D2F9C8", "#A3EE91", "#69DA4D", "#42962E", "#0E2D06"]
+              span(class='mx-auto rounded-sm h-4 w-4' style=`background-color: ${GREENS[Math.min(dayData.many, 4)]}`)
```

Comment:
Just a small note: having inline style is generally discouraged and should be avoided. I this case we can either use the green colors from the tailwind palette, or extend it and use those in our code.

Reply:
I tried using the tailwind colors, however the build step seems to have optimized out the required styles.

Reply:
In that case you can whitelist the classnames in the tailwind config: https://tailwindcss.com/docs/optimizing-for-production#purge-css-options

Reply:
When #597 gets merged we can also try to list the classnames instead of the hex values in the template as a JS array and maybe we won't need the safelist.

Reply:
We couldn't really solve it using tailwindcss classes so some help would be handy @OmTheTurtle

Reply:
We're using now the tailwind bg classes, but they have a very distingished blue hue to the green color. So if you could help us with using postcss config to use the hardcoded colors, we would thank you for that

Reply:
Nvm we solved it